x-pack/plugins/security_solution/public/common/store/store.ts

```diff
@@ -80,7 +80,8 @@ export const createStore = (
     )
   );
 
-  epicMiddleware.run(createRootEpic<State>());
+  // @ts-expect-error
```

Comment:
Le's create an issue here to nor forget!

Reply:
not to worry about it, @oatkiller tried to fix it but was never able to reproduce it. So I think we can remove the `ts-ignore` in a follow-up PR

Reply:
This error was actually an artifact of the optimized (for editing) `tsconfig`, which was generated by running:

```
node x-pack/plugins/security_solution/scripts/optimize_tsconfig
```

After returning to the default (checked-in) `tsconfig` via:

```
node x-pack/plugins/security_solution/scripts/unoptimize_tsconfig.js
```

it passes the type checker without error, so I'm removing the `@ts-ignore`